convert_util.c

```diff
@@ -109,9 +109,11 @@ convert_parse_tlvs(const uint8_t *buff, size_t buff_len,
 			    CONVERT_ALIGN(sizeof(*conv_connect)))
 				return -1;
 
-			opts->flags		|= CONVERT_F_CONNECT;
-			opts->remote_addr	= conv_connect->remote_addr;
-			opts->remote_port	= conv_connect->remote_port;
+			opts->flags			|= CONVERT_F_CONNECT;
```

Comment:
also in the right order?

Reply:
This is on a single byte (uint8_t flags), thus no need to worry